src/settings/JobProfiles/ViewJobProfile.js

```diff
@@ -261,14 +245,16 @@ export class ViewJobProfile extends Component {
       );
     }
 
-    // TODO: UIDATIMP-180 - use real IDs
+    // start
+    // TODO: use real IDs when sample data will be removed
```

Comment:
Todo description is not enough for my point of view. Add more context of what exactly should be done.